algorithms/binary_search/python/v_chmykov.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+
+def binary_search(array, number):
+    length = len(array)
+    if length == 0:    
+        return False
+
+    low = 0
+    high = length - 1
+    while low <= high:
+        mid = (low + high) // 2
+        if number < array[mid]:
+            high = mid - 1
+        elif number > array[mid]:
+            low = mid + 1
+        else:
+            return mid
+    else:
+        False
```

Comment:
Please give a `return` statement here. This is not good coding practice.

Reply:
Sorry, my bad, I wanted to add this but forgot. Fixed it